maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -282,9 +299,13 @@ data class AssertCommand(
     val visible: ElementSelector? = null,
     val notVisible: ElementSelector? = null,
     val timeout: Long? = null,
+    val label: String? = null,
 ) : Command {
 
     override fun description(): String {
+        if (label != null && label != ""){
```

Comment:
```suggestion
        if (label != null){
```

See my comment on `Condition.kt`